scripts/enable_eth_group_mdev.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+#N3000 Eth group mdev
+set -m
+
+pcie_count=$(ls /sys/bus/pci/devices/*/device | wc -l)
+if [[ $pcie_count -lt 1 ]]; then
+    echo "Failed to locate drivers."
+    exit 1
+fi
+echo -e "PCIe device count:$pcie_count"
+
+
+#Bus number
+bus_num=$1
+PCIE_DEVICES=/sys/bus/pci/devices/*$bus_num*/device
+#echo $PCIE_DEVICES
+for file in $PCIE_DEVICES
+do
+    #echo -e "Device $file\n"
+    device_id=$(cat $file)
+    #echo $device_id
+    if [[ $device_id == 0x0b30 ]]; then
+     break
+    fi
+
+done
+
+echo "Found fpga N3000 device $file"
+
+
+FPGA_FEATURID=/sys/bus/pci/devices/*$bus_num*/fpga_region/region*/dfl-fme*/dfl-fme*/feature_id
+echo " Feature: $FPGA_FEATURID"
+feature_list=()
+for file in $FPGA_FEATURID
+do
+    echo "Feaure id $file"
+    feature_id=$(cat $file)
+    echo "Feature id: $feature_id"
+	if [[ $feature_id != 0x10 ]]; then
```

Comment:
What does the 0x10 stand for?  Will it ever change?

Reply:
Eth group DFH feature ID for FPGA N3000